book/website/communication/research-objects.md

```diff
@@ -0,0 +1,37 @@
+(cm-ro)=
+# Research Object to capture the Research Life Cycle
+
+## Prerequisites / recommended skill level
+
```

Comment:
```suggestion
No previous knowledge is needed.
```

Reply:
Or maybe actually refer people to information on metadata, since you use this in the motivation below?